app/i18n/locales/zh-Hant.json

```diff
@@ -122,6 +122,10 @@
   "settings.general.aboutYoroi.website": "Yoroi網站",
   "settings.general.aboutYoroi.youtube": "EMURGO YouTube",
   "settings.general.languageSelect.label": "語言",
+  "settings.general.languageSelect.labelInfo": "The selected language translation is fully provided by the community",
```

Comment:
FYI you should only ever modify the English file in a PR. Otherwise it may cause merge conflicts with the translation system we use.

Reply:
Good to know! Thanks